scss/_variables.scss

```diff
@@ -1156,8 +1156,9 @@ $form-range-thumb-border-radius:           50% !default;
 $form-range-thumb-box-shadow:              0 .1rem .25rem rgba($black, .1) !default;
 $form-range-thumb-focus-box-shadow:        null !default;
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in Edge
-$form-range-thumb-active-bg:               null !default;
-$form-range-thumb-active-border:           $component-active-bg !default; // Boosted mod
+$form-range-thumb-hover-bg:                $black !default; // Boosted mod
+$form-range-thumb-active-bg:               $primary !default; // Boosted mod
+$form-range-thumb-active-border:           $border-width solid $primary !default; // Boosted mod
 $form-range-thumb-disabled-bg:             $gray-500 !default;
 $form-range-thumb-transition:              background-color $transition-duration $transition-timing, border-color $transition-duration $transition-timing, box-shadow $transition-duration $transition-timing !default;
```

Comment:
Should we keep all those transitions ? (Not sure what'd be the best here)

Reply:
I would say yes, let's keep them because they were already there and are not part of the points to be change in this PR.